scripts/irods/test/test_iput.py

```diff
@@ -946,6 +947,8 @@ def test_server_honors_server_config_option_checksum_read_buffer_size_in_bytes__
             # self.object_name deleted in teardown
             os.unlink(local_file)
 
+    # TODO Open issue for this.
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Needs to be updated for topology testing or a new test for topology testing needs to be added")
     def test_server_rejects_invalid_server_config_options_for_checksum_read_buffer_size_in_bytes__issue_7947(self):
```

Comment:
Handle TODO.

When testing in a topology, we don't have a way to tell the other computer to reload its configuration.

This is not a blocker for this PR.

Reply:
Linked to #6835.